.github/workflows/umd-size-checker.yml

```diff
@@ -1,15 +0,0 @@
-name: UMD size checker
-
-on: [pull_request]
-
-jobs:
-  check:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: preactjs/compressed-size-action@v2
-        with:
-          repo-token: '${{ secrets.GITHUB_TOKEN }}'
-          build-script: 'prepublishOnly'
-          pattern: './packages/*/dist/*.{js,css,json}'
-          exclude: '{**/*.html,**/*.map,**/node_modules/**}'
```

Comment:
Why removing this?

Reply:
moved to global lerna-ci, no need to be outside of it